Benchmarks/Models.swift

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-let benchmarkModels: [String: BenchmarkModel] = [
-    "LeNetMNIST": LeNetMNIST(),
-    "ResNetCIFAR10": ResNetCIFAR10(),
+let benchmarkModels: [String: BenchmarkModel.Type] = [
+    LeNetMNIST.name: LeNetMNIST.self,
+    ResNetCIFAR10.name: ResNetCIFAR10.self,
```

Comment:
If there's no need for flexibly mapping from name to `BenchmarkModel.Type`, what do you think of something like the following:

```suggestion
let benchmarkModels = Dictionary([LeNetMNIST.self, ResNetCIFAR10.self].map { ($0.name, $0) })
```

Note: I assume instantiating these `BenchmarkModel` types is expensive, and so that's why we're using their metatypes? Or is there some other reason?